private/bufpkg/bufplugin/bufplugin.go

```diff
@@ -90,6 +93,28 @@ func PluginToProtoPluginRegistryType(plugin Plugin) registryv1alpha1.PluginRegis
 	return registryType
 }
 
+// LanguagesToProtoLanguages determines the appropriate registryv1alpha1.PluginRegistryType for the plugin.
+func LanguagesToProtoLanguages(languages []string) []registryv1alpha1.PluginLanguage {
+	languageToEnum := make(map[string]registryv1alpha1.PluginLanguage)
+	for pluginLanguageKey, pluginLanguage := range registryv1alpha1.PluginLanguage_value {
+		pluginLanguageKey := strings.TrimPrefix(pluginLanguageKey, "PLUGIN_LANGUAGE_")
```

Comment:
This might be considered brittle, and we might want to have an explicit string to proto map. But I'd argue the enum name is unlikely to change and it avoids having to remember to update a map somewhere.

Also, this doesn't fail with an error if the language is unknown. We probably don't want to block plugin uploads because a language isn't supported? Although we could just omit the language until it's added and be more explicit here.